src/inc/msquic.h

```diff
@@ -946,6 +946,16 @@ void
 #endif
 #define QUIC_PARAM_GLOBAL_TLS_PROVIDER                  0x0100000A  // QUIC_TLS_PROVIDER
 #define QUIC_PARAM_GLOBAL_STATELESS_RESET_KEY           0x0100000B  // uint8_t[] - Array size is QUIC_STATELESS_RESET_KEY_LENGTH
+#define QUIC_PARAM_GLOBAL_STATELESS_RETRY_CONFIG        0x0100000C  // QUIC_STATELESS_RETRY_CONFIG
```

Comment:
We also need to support registry based configuration. Do you plan to add that to this PR or a follow up?

Reply:
I'll add it in this PR.